bpf/lib/egress_policies.h

```diff
@@ -16,6 +16,11 @@
 #define EGRESS_STATIC_PREFIX (sizeof(__be32) * 8)
 #define EGRESS_PREFIX_LEN(PREFIX) (EGRESS_STATIC_PREFIX + (PREFIX))
 #define EGRESS_IPV4_PREFIX EGRESS_PREFIX_LEN(32)
+/* These are special IP values in the CIDR 0.0.0.0/8 range that map to specific
+ * case for in the egress gateway policies handling.
+ */
+#define EGRESS_GATEWAY_NO_GATEWAY (0)
+#define EGRESS_GATEWAY_EXCLUDED_CIDR (1)
```

Comment:
I think this needs to be:

```suggestion
#define EGRESS_GATEWAY_EXCLUDED_CIDR bpf_htonl(1)
```

as the gateway IP is encoded as BE in the policy map

Reply:
Oh yes indeed good catch, thanks :).